Tasks/PullRequestCodeAnalysis/task.json

```diff
@@ -0,0 +1,43 @@
+{
+    "id": "683B88A0-96C3-11E6-8625-95B67567393D",
+    "name": "PullRequestCodeAnalysis",
+    "friendlyName": "Pull Request Code Analysis",
+    "description": "Post code analysis issues as comments on a pull request",
+    "helpMarkDown": "[More Information](   NEEDS LINK    )",
+    "category": "Build",
+    "visibility": [
+        "Build"
+    ],
+    "author": "Microsoft Corporation",
+    "demands": [],
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 0
+    },
+    "minimumAgentVersion": "1.99.0",
```

Comment:
I'm not sure either, but every task.json has this. It still builds without, though, so let's remove it and see.